lib/rules/selector-pseudo-class-case/index.js

```diff
@@ -61,6 +61,19 @@ const rule = function(expectation) {
             return;
           }
 
+          if (context.fix) {
+            if (rule.raws.selector) {
+              rule.raws.selector.raw = rule.raws.selector.raw.replace(
```

Comment:
I found a problem with the CSS as follow.

```css
a::FIRST-LETTER, a:FIRST {color: pink;}
```

The result of fixed this is as follow.

```css
a::first-LETTER, a:FIRST {color: pink;}
```

To improve this, I think we need to change this as follows.

```js
pseudoNode.value = expectedPseudo;
```

Then we process the result of parseSelector as follows.

```js
const fixedSelector = parseSelector(rule.raws.selector ? rule.raws.selector.row : rule.selector, ...
//...

if (context.fix) {
  if (rule.raws.selector) {
    rule.raws.selector.raw = fixedSelector
  } else {
    rule.selector = fixedSelector;
  }
}
```

Reply:
Hi, thank you for spotting this case! I have updated the PR